git-toolfile.spec

```diff
@@ -7,7 +7,7 @@ Requires: git
 %install
 
 case "%{cmsplatf}" in
-  slc6*|fc*)
+  slc7*|slc6*|fc*)
```

Comment:
Same here. Make Linux default and keep the current default for OS X, if it actually applies.